server/src/test/scala/services/LhrRedListDatesImplSpec.scala

```diff
@@ -13,5 +13,51 @@ class LhrRedListDatesImplSpec extends Specification {
     "Give the T4 opening date as 2021-06-29 in BST" >> {
       LhrRedListDatesImpl.t4RedListOpeningDate === SDate("2021-06-29T00:00", Crunch.europeLondonTimeZone).millisSinceEpoch
     }
+    "Give the Red list start date as 2021-02-21 in BST" >> {
+      LhrRedListDatesImpl.startRedListingDate === SDate("2021-02-15T00:00", Crunch.europeLondonTimeZone).millisSinceEpoch
+    }
+    "Give the Red list end date as 2021-12-21 in BST" >> {
+      LhrRedListDatesImpl.endRedListingDate === SDate("2021-12-15T00:00", Crunch.europeLondonTimeZone).millisSinceEpoch
+    }
+  }
+
+  "A LhrRedListDatesImpl.dayHasPaxDiversions should" >> {
+    "Give a date it should be red listed as true if it is after the start date and before the end date" >> {
+      LhrRedListDatesImpl.isRedListActive(SDate("2021-06-01T00:00", Crunch.europeLondonTimeZone)) === true
+    }
+    "Give a date it should be red listed as false if it is before the start date" >> {
+      LhrRedListDatesImpl.isRedListActive(SDate("2021-02-14T00:00", Crunch.europeLondonTimeZone)) === false
+    }
+    "Give a date it should be red listed as false if it is after the end date" >> {
+      LhrRedListDatesImpl.isRedListActive(SDate("2021-12-16T00:00", Crunch.europeLondonTimeZone)) === false
+    }
   }
+
+  "LhrRedListDatesImpl.overlapsRedListDates should" >> {
+    "Give a start and end dates for multi days containing red List active range, it should be overlaps red listed dates as true" >> {
+      LhrRedListDatesImpl.overlapsRedListDates(SDate("2021-01-01T00:00", Crunch.europeLondonTimeZone),
+        SDate("2022-01-15T00:00", Crunch.europeLondonTimeZone)) === true
+    }
+
+    "Give a start and end dates not containing red List active range, it should be overlaps red listed dates as false" >> {
+      LhrRedListDatesImpl.overlapsRedListDates(SDate("2021-01-01T00:00", Crunch.europeLondonTimeZone),
+        SDate("2021-02-14T00:00", Crunch.europeLondonTimeZone)) === false
+    }
+
+    "Give a start and end dates with red List active range, it should be overlaps red listed dates as true" >> {
+      LhrRedListDatesImpl.overlapsRedListDates(SDate("2021-05-01T00:00", Crunch.europeLondonTimeZone),
+        SDate("2021-06-14T00:00", Crunch.europeLondonTimeZone)) === true
+    }
+
+    "Give a start date with in red list active range , it should be overlaps red listed dates as true" >> {
+      LhrRedListDatesImpl.overlapsRedListDates(SDate("2021-01-30T00:00", Crunch.europeLondonTimeZone),
+        SDate("2022-01-01T00:00", Crunch.europeLondonTimeZone)) === true
+    }
+
+    "Give a end date with in red list active range , it should be overlaps red listed dates as true" >> {
+      LhrRedListDatesImpl.overlapsRedListDates(SDate("2021-01-01T00:00", Crunch.europeLondonTimeZone),
+        SDate("2021-02-24T00:00", Crunch.europeLondonTimeZone)) === true
+    }
```

Comment:
These tests are great. It would be good to try making the descriptions easier to read a little, eg
```
  "return true when given a date falling inside the red list period"

  "return true given a date range overlapping the start of the red list period" >> {

  "return true given a date range overlapping the end of the red list period" >> {

  "return true given a date range falling inside the red list period"

  "return true given a date range starting before and ending after the red list period"

  "return false given a date range falling starting after the redlist period ends"

  "return false given a date range falling ending before the redlist period starts"
```